<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightClickMenuDialog</class>
 <widget class="QDialog" name="RightClickMenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>87</width>
    <height>148</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="transferBtn">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>30</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(48,52,72);color:rgb(192,196,212);border:0px solid rgb(239,239,239);}
QPushButton:hover{background-color:rgb(44,48,67);color:white;}</string>
   </property>
   <property name="text">
    <string>发起转账</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renameBtn">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>59</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(48,52,72);color:rgb(192,196,212);border:0px solid rgb(239,239,239);}
QPushButton:hover{background-color:rgb(44,48,67);color:white;}</string>
   </property>
   <property name="text">
    <string>修改账户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>88</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(48,52,72);color:rgb(192,196,212);border:0px solid rgb(239,239,239);}
QPushButton:hover{background-color:rgb(44,48,67);color:white;}</string>
   </property>
   <property name="text">
    <string>导出私钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyBtn">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(48,52,72);color:rgb(192,196,212);border:0px solid rgb(239,239,239);}
QPushButton:hover{background-color:rgb(44,48,67);color:white;}</string>
   </property>
   <property name="text">
    <string>复制地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>117</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:rgb(48,52,72);color:rgb(192,196,212);border:0px solid rgb(239,239,239);}
QPushButton:hover{background-color:rgb(44,48,67);color:white;}</string>
   </property>
   <property name="text">
    <string>删除账户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
